pandas/tests/io/test_sql.py

```diff
@@ -28,7 +28,7 @@
 import pytest
 
 import pandas.compat as compat
-from pandas.compat import PY2, PY36, lrange, range, string_types
+from pandas.compat import PY2, PY35, PY36, lrange, range, string_types
```

Comment:
really really don't change things, if you are forcing chanegs then roll back the env changes which cause this.